src/components/popup/PopupBody.tsx

```diff
@@ -1,8 +1,13 @@
 import * as React from 'react';
 import { StyledPopupBody } from './style';
 
-const PopupBody = ({ ...props }: React.HTMLAttributes<HTMLDivElement>): JSX.Element => (
-  <StyledPopupBody className="popupBody" {...props} />
+export interface PopupBodyProps extends React.HTMLAttributes<HTMLDivElement> {
+  children: React.ReactNode;
+  autofocusInside?: boolean;
+}
+
+const PopupBody = ({ autofocusInside = true, ...props }: PopupBodyProps): JSX.Element => (
+  <StyledPopupBody data-autofocus-inside={autofocusInside || undefined} className="popupBody" {...props} />
```

Comment:
@vozbrann What's the purpose for this undefined?

Reply:
`autofocusInside` will be undefined by default, no need to add `||`

Reply:
@vlacher12 `autofocusInside = true` should be `true` by default

Reply:
I mean, to replace 
`<StyledPopupBody data-autofocus-inside={autofocusInside || undefined} className="popupBody" {...props} />`
by
`<StyledPopupBody data-autofocus-inside={autofocusInside} className="popupBody" {...props} />`

Reply:
`autofocusInside = true` still will be `true` by default

Reply:
`|| undefined` is just redundant, isn't it?

Reply:
@vlacher12 
`data-autofocus-inside={false}` is not working. Focus is still there
`data-autofocus-inside={undefined}` works